examples/llama2/user_script.py

```diff
@@ -60,7 +61,7 @@ def get_merged_sample_with_past_kv_inputs(
     elif model.hf_config is not None:
         config = model.get_hf_model_config() if model else LlamaConfig.from_pretrained(model_id)
     else:
-        config = model.model_attributes
+        config = Namespace(**model.model_attributes)
```

Comment:
add comments why need the namespace and what's the problem of original calls.

Reply:
Comments updated. The config variable is used via config.item, hf_config can be used via this but when custom models it is saved in model_attributes dict. I create a Namespace class here to use it via config.item rather than config["item"].

Reply:
nit: https://docs.python.org/3/library/types.html#types.SimpleNamespace
there is a `SimpleNamespace` in python `types` could achieve the same thing.